### ISSUE TITLE
Issues with Mongo 5.0

### ISSUE THREAD

**Issue by Guillermogsjc** [idealo/mongodb-slow-operations-profiler#18]
Hi @kagahd ,

congratulations for the super useful tool. 

We have several mongos, the tools works very nice against Mongo 4.2 ones, but crashes in the server info internal call on the Mongo 5.0 ones. It throws messages like:

```shell
ExecutionException while getting updated server status within 5000 ms for 'dev-mongo-5-arm-local' at
```

And the collector is not correctly taking slowop logs from the database.

Any idea on this? Maybe it would be a little change in the server information call.

Regards

**Comment by kagahd** [comment]
Hi @Guillermogsjc , sorry for my late reply - I just found it today. I'll look into it asap and report back to you.

**Comment by kagahd** [comment]
Thanks for the bug report @Guillermogsjc ! 
I just released a new version ([3.2.2](https://github.com/idealo/mongodb-slow-operations-profiler/releases/tag/v3.2.2)) which fixes it. Please close this issue if it works for you or report back. Thanks!

**Comment by Guillermogsjc** [comment]
Thank you @kagahd  for this great profiler
